bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -340,6 +345,53 @@ impl Room {
         });
     }
 
+    pub fn create_poll(
+        &self,
+        question: String,
+        answers: Vec<String>,
+        max_selections: u64,
+        poll_kind: PollKind,
+        txn_id: Option<String>,
+    ) -> Result<(), ClientError> {
+        let timeline = match &*RUNTIME.block_on(self.timeline.read()) {
+            Some(t) => Arc::clone(t),
+            None => {
+                return Err(anyhow!("Timeline not set up, can't send the poll").into());
+            }
+        };
+
+        let poll_answers_vec: Vec<UnstablePollAnswer> = answers
+            .iter()
+            .enumerate()
+            .map(|(index, option)| UnstablePollAnswer::new(index.to_string(), option))
```

Comment:
@jplatte is it ok to use the [uuid crate](https://docs.rs/uuid/latest/uuid/) for assigning random IDs here?
It may be better considering that "poll start" events can be edited.

Reply:
Added random IDs [here](https://github.com/matrix-org/matrix-rust-sdk/pull/2391/commits/a46cc77f44d557a97fff63061da1f74a5949cf72). Having random IDs is more robust against the edit of the "poll start" event